src/js/repeat.js

```diff
@@ -282,6 +283,15 @@ export default {
 
         // Determine the index of the repeat series.
         let repeatSeriesIndex = this.getIndex( repeatInfo );
+
+        // get repeatSeriesIndex using parent repeat element
+        // this fixes a nested repeats error https://github.com/enketo/enketo-core/issues/720
+        if( repeatPath && repeatPath.split( '/' ).length - 1 > 3 && repeatInfo ){
```

Comment:
So, if I understand correctly the bug starts with triple-nested repeats (and continues in quadruple-nested repeats etc). Is that right?

Isn't the actual bug in `this.getIndex`?  If so, would it be best to fix that function instead of working around it for more-than-double-nested repeats, or am I misunderstanding?

Reply:
makes sense; going to move this code into `this.getIndex`